docs/theming.md

```diff
@@ -0,0 +1,126 @@
+# Theming your Angular Material app
+
+
+### What is a theme?
+A **theme** is the set of colors that will be applied to the Angular Material components. The
+library's approach to theming is based on the guidance from the [Material Design spec][1]. 
+
+In Angular Material, a theme is created by composing multiple palettes. In particular, 
+a theme consists of:
+* A primary palette: colors most widely used across all screens and components.
+* An accent palette: colors used for the floating action button and interactive elements.
+* A "warn" palette: colors used to convey error state.
```

Comment:
"Primary palette" and "accent palette" both sounded natural, while "warn palette" seemed weird.